external-crates/move/crates/move-vm-runtime/src/jit/execution/ast.rs

```diff
@@ -864,24 +872,12 @@ impl Function {
     }
 
     pub fn pretty_string(&self) -> String {
-        let id = &self.module;
-        format!(
-            "0x{}::{}::{}",
-            id.address(),
-            id.name().as_str(),
-            self.name.as_str()
-        )
+        self.name.to_string().unwrap()
     }
 
     #[cfg(any(debug_assertions, feature = "tracing"))]
```

Comment:
Just a note (nothing for you to do): I'm going to be removing this cfg in another PR here soon -- runs afoul of building in release mode without tracing enabled since this function is used in the `Debug` implementation for bytecode.